main.go

```diff
@@ -51,6 +52,8 @@ func main() {
 		namespaceController := namespace.NewNamespaceController(k8sClientset, calicoClient)
 		go namespaceController.Run(5, reconcilerPeriod, stop)
 	case "policy":
+	    policyController := networkpolicy.NewPolicyController(k8sClientset, calicoClient)
+		go policyController.Run(5, reconcilerPeriod, stop)
```

Comment:
Making a note that we should think about the number we're setting for each controller's worker count. 

I'm not sure 5 makes sense for all of them.

Reply:
For now, I am setting up to bellow number of worker threads by default. (If not supplied by user)
- workloadEndpoint    3
- policies  1
- profiles 1 

Pods being frequently deployed/scheduled, we may have higher number of threads (3 here) and policies/profiles being less frequently updated we may have single worker thread by default. 
Maybe going ahead we can tune as required depending on our observations.

If you have any other suggestion then please let me know.